### ISSUE TITLE
Allow forwarding CLI options

### ISSUE THREAD

**Issue by jcrist** [dask/dask-yarn#69]
Some options on the distributed CLI should be settable via dask-yarn (e.g. `--dashboard-address`). There are a few ways we could accomplish this:

- Update distributed so that all parameters are also part of the configuration. Users could then provide them via `env` (or via configuration #42).
- Add an option to specify these parameters as part of the command

From https://github.com/jcrist/skein/issues/176.

**Comment by mrocklin** [comment]
I think that it would be useful to start thinking about a uniform way to
specify things across all of the projects.

**Comment by jcrist** [comment]
I agree. I think a precursor to this would be making the CLI scripts in distributed more uniform in terms of argument parsing/config, and how processes are started/stopped.

*Edit: misread your comment. I still think the above is important for yarn specific (since we need our own cli scripts), but common arguments for the cluster managers for specifying the worker/scheduler command would be good.*
